### PR TITLE
Master misc fix ged

### PR DESCRIPTION
**Pull request by ged-odoo**
Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by sdegueldre** [comment]
@robodoo rebase-ff

File: addons/web/static/src/views/fields/fields.scss [hunk 1]
```diff
@@ -1,3 +1,7 @@
 .o_field_cursor_disabled {
     cursor: not-allowed;
 }
+
+.o_field_widget {
+    display: contents;
+}
```
Comment (sdegueldre) [inline addons/web/static/src/views/fields/fields.scss:7]: Missing newline

File: addons/website/static/src/js/backend/redirect_field.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+/** @odoo-module **/
+
+import { _lt } from "@web/core/l10n/translation";
+import { registry } from "@web/core/registry";
+
+const { Component } = owl;
+
+class RedirectField extends Component {
+    get info() {
+        return this.props.value ? _lt("Published") : _lt("Unpublished");
```
Comment (sdegueldre) [inline addons/website/static/src/js/backend/redirect_field.js:10]: Why `_lt`?

File: addons/website/static/src/js/backend/redirect_field.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+    }
+
+    onClick() {
+        this.env.onClickViewButton({
+            clickParams: {type: "object", name: "open_website_url"},
```
Comment (sdegueldre) [inline addons/website/static/src/js/backend/redirect_field.js:15]: You don't have prettier in website ;)

**Review by sdegueldre — APPROVED** [review]

**Review by aab-odoo — APPROVED** [review]
robodoo r+

**Review by aab-odoo — APPROVED** [review]

**Comment by kebeclibre** [comment]
robodoo r+

**Comment by aab-odoo** [comment]
robodoo r+

**Comment by aab-odoo** [comment]
robodoo retry

**Pull request closed without merging**